components/hdw-spiffs/spiffs_json.c

```diff
@@ -23,7 +22,7 @@ char * loadJson(char * name)
     size_t sz;
     if(!spiffsReadFile(name, &buf, &sz))
     {
-        ESP_LOGE("JSON", "Failed to read %s", name);
+        printf("Failed to read %s", name);
```

Comment:
Any particular reason you replaced `ESP_LOGE()` with `printf()`? My intent was to turn these all off in final firmware via `sdkconfig`

Reply:
EMU/Non-EMU operation.  How else would you want to handle it?

Reply:
The emu should already route those prints to printf anyway...